engine/src/main/java/org/hibernate/search/engine/search/predicate/spi/MatchIdPredicateBuilder.java

```diff
@@ -0,0 +1,13 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.engine.search.predicate.spi;
+
+public interface MatchIdPredicateBuilder<B> extends SearchPredicateBuilder<B> {
+
+	void value(Object values);
```

Comment:
`values` should be singular here (`value`), I think? Or did I misunderstand how it works?

Reply:
no, it's a typo